workflow-templates/db-bundle.yml

```diff
@@ -0,0 +1,17 @@
+name: db-bundle
+
+on:
+  workflow_dispatch:
+  push:
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  bundle:
+    uses: TAG-APAC-Division/platform-reusable-workflows/.github/workflows/db-bundle.yml@v0.21.2
+    secrets: inherit
```

Comment:
@lorenzo-kun could you please replace this with explicit secret definitions for clarity? Thanks.

Reply:
Sure, done 👍